python/cugraph-service/client/pyproject.toml

```diff
@@ -43,3 +44,8 @@ version = {file = "cugraph_service_client/VERSION"}
 include = [
     "cugraph_service_client",
 ]
+
+[tool.rapids-build-backend]
+build-backend = "setuptools.build_meta"
+dependencies-file = "../../../dependencies.yaml"
+disable-cuda = true
```

Comment:
I just added this.

`cugraph-service-client` doesn't have any CUDA-specific dependencies and we don't publish any wheels for it, so it should always have `disable-cuda=true`.